src/cron/syncSubscriptionWithCIO.ts

```diff
@@ -46,7 +46,7 @@ const cron: Cron = {
         break;
       }
     }
-    logger.info(`synced subscriptions for ${syncedSubscriptions} users`);
+    logger.info({ count: syncedSubscriptions }, 'synced subscriptions');
```

Comment:
We can also change the above `debug` call to the same pattern then.

Reply:
Done